simplq/src/components/pages/Admin/index.jsx

```diff
@@ -35,9 +33,8 @@ export default (props) => {
 
   const update = useCallback(() => {
     clearTimeout(timeoutId);
-    requestMaker(QueueRequestFactory.get(queueId)).then((data) => {
+    requestMaker(getActiveTokens(queueId)).then((data) => {
       if (data) {
-        setTokens(data.tokens);
         setQueueName(data.queueName);
```

Comment:
Still figuring out the best way to change this part.

Reply:
@daltonfury42 solved this in `store/asyncActions/getToken.js`. You can also use `useEffects()` in hooks. Since async action is already in hook, you can use `useEffects()` to deal with timers. `AdminPage` itself should just connect to the store. Wrap components that depend on async action in `LoadingStatus` component.